kerl

```diff
@@ -259,8 +341,8 @@ fi
 if [ "$KERL_BUILD_BACKEND" = 'git' ]; then
     KERL_USE_AUTOCONF=1
 elif [ "$KERL_BUILD_BACKEND" != 'tarball' ]; then
-    l=e stderr "Unhandled value KERL_BUILD_BACKEND='${KERL_BUILD_BACKEND}'"
-    l=t stderr "KERL_BUILD_BACKEND must be one of 'git' (default) or 'tarball'"
+    error "enhandled value KERL_BUILD_BACKEND=${KERL_BUILD_BACKEND}."
```

Comment:
Typo here

Reply:
Nice catch. I'm gonna check this is CSpell.

Reply:
5f5f773